ractor/src/pg/mod.rs

```diff
@@ -224,6 +412,37 @@ pub fn get_members(group_name: &GroupName) -> Vec<ActorCell> {
 ///
 /// Returns a [`Vec<GroupName>`] representing all the registered group names
 pub fn which_groups() -> Vec<GroupName> {
+    let monitor = get_monitor();
+    monitor
+        .map
+        .iter()
+        .map(|kvp| kvp.key().clone())
+        .map(|(_scope, group)| group.clone())
```

Comment:
groups can be in multiple scopes I think, we probably need to de-duplicate this list